Doesn't compile

After 1d92a7064a7d939b654430a71521e8113d47e3b1 (lol). I'm using gcc 15.2.1

```
[  1%] Building CXX object lib/KdLib/test/CMakeFiles/KdLibTest.dir/src/ranges/tst_zip_view.cpp.o
cd /home/herman/prog/build/TrenchBroom/build/lib/KdLib/test && /usr/bin/c++ -DCATCH_CONFIG_ENABLE_ALL_STRINGMAKERS=1 -I/home/herman/prog/build/TrenchBroom/lib/KdLib/include -isystem /home/herman/prog/build/TrenchBroom/build/vcpkg_installed/x64-linux/include -O3 -DNDEBUG -std=c++20 -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -pedantic -Wshadow=local -Wnon-virtual-dtor -Wold-style-cast -Wcast-align -Wunused -Wconversion -Wsign-conversion -Wmisleading-indentation -Wduplicated-cond -Wduplicated-branches -Wlogical-op -Wformat=2 -Wimplicit-fallthrough -O3 -MD -MT lib/KdLib/test/CMakeFiles/KdLibTest.dir/src/ranges/tst_zip_view.cpp.o -MF CMakeFiles/KdLibTest.dir/src/ranges/tst_zip_view.cpp.o.d -o CMakeFiles/KdLibTest.dir/src/ranges/tst_zip_view.cpp.o -c /home/herman/prog/build/TrenchBroom/lib/KdLib/test/src/ranges/tst_zip_view.cpp
In file included from /home/herman/prog/build/TrenchBroom/lib/KdLib/test/src/ranges/tst_zip_view.cpp:21:
/home/herman/prog/build/TrenchBroom/lib/KdLib/include/kd/ranges/zip_view.h: In instantiation of ‘kdl::ranges::zip_view<std::ranges::ref_view<const std::vector<int, std::allocator<int> > >, std::ranges::ref_view<const std::array<float, 3> > >::size() const requires (sized_range<Views> && ...)::<lambda(auto:21&& ...)> [with auto:21 = {const std::ranges::ref_view<const std::vector<int, std::allocator<int> > >&, const std::ranges::ref_view<const std::array<float, 3> >&}]’:
/usr/lib/gcc/x86_64-pc-linux-gnu/15/include/g++-v15/type_traits:3333:30:   required from ‘struct std::is_nothrow_invocable<kdl::ranges::zip_view<std::ranges::ref_view<const std::vector<int, std::allocator<int> > >, std::ranges::ref_view<const std::array<float, 3> > >::size() const requires (sized_range<Views> && ...)::<lambda(auto:21&& ...)>, const std::ranges::ref_view<const std::vector<int, std::allocator<int> > >&, const std::ranges::ref_view<const std::array<float, 3> >&>’
 3333 |     : public __bool_constant<__is_nothrow_invocable(_Fn, _ArgTypes...)>
      |                              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/lib/gcc/x86_64-pc-linux-gnu/15/include/g++-v15/tuple:2912:37:   required from ‘constexpr const bool std::__unpack_std_tuple<template<class _Fn, class ... _ArgTypes> struct std::is_nothrow_invocable, kdl::ranges::zip_view<std::ranges::ref_view<const std::vector<int, std::allocator<int> > >, std::ranges::ref_view<const std::array<float, 3> > >::size() const requires (sized_range<Views> && ...)::<lambda(auto:21&& ...)>, const std::tuple<std::ranges::ref_view<const std::vector<int, std::allocator<int> > >, std::ranges::ref_view<const std::array<float, 3> > >&>’
 2912 |       = _Trait<_Tp, const _Up&...>::value;
      |                                     ^~~~~
/usr/lib/gcc/x86_64-pc-linux-gnu/15/include/g++-v15/tuple:2931:14:   required from ‘constexpr decltype(auto) std::apply(_Fn&&, _Tuple&&) [with _Fn = kdl::ranges::zip_view<ranges::ref_view<const vector<int, allocator<int> > >, ranges::ref_view<const array<float, 3> > >::size() const requires (sized_range<Views> && ...)::<lambda(auto:21&& ...)>; _Tuple = const tuple<ranges::ref_view<const vector<int, allocator<int> > >, ranges::ref_view<const array<float, 3> > >&]’
 2931 |     noexcept(__unpack_std_tuple<is_nothrow_invocable, _Fn, _Tuple>)
      |              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/herman/prog/build/TrenchBroom/lib/KdLib/include/kd/ranges/zip_view.h:410:22:   required from ‘constexpr auto kdl::ranges::zip_view<Views>::size() const requires (sized_range<Views> && ...) [with Views = {std::ranges::ref_view<const std::vector<int, std::allocator<int> > >, std::ranges::ref_view<const std::array<float, 3> >}]’
  410 |     return std::apply(
      |            ~~~~~~~~~~^
  411 |       [](auto&&... b) { return std::min({std::ranges::size(b)...}); }, mBase);
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/herman/prog/build/TrenchBroom/lib/KdLib/test/src/ranges/tst_zip_view.cpp:171:7:   required from here
  171 |       CHECK(z.size() == 3);
      |             ~~~~~~^~
/home/herman/prog/build/TrenchBroom/lib/KdLib/include/kd/ranges/zip_view.h:411:40: error: no matching function for call to ‘min(<brace-enclosed initializer list>)’
  411 |       [](auto&&... b) { return std::min({std::ranges::size(b)...}); }, mBase);
      |                                ~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/herman/prog/build/TrenchBroom/lib/KdLib/include/kd/ranges/zip_view.h:411:40: note: there are 2 candidates
In file included from /usr/lib/gcc/x86_64-pc-linux-gnu/15/include/g++-v15/bits/hashtable_policy.h:36,
                 from /usr/lib/gcc/x86_64-pc-linux-gnu/15/include/g++-v15/bits/hashtable.h:37,
                 from /usr/lib/gcc/x86_64-pc-linux-gnu/15/include/g++-v15/bits/unordered_map.h:33,
                 from /usr/lib/gcc/x86_64-pc-linux-gnu/15/include/g++-v15/unordered_map:43,
                 from /usr/lib/gcc/x86_64-pc-linux-gnu/15/include/g++-v15/functional:65,
                 from /home/herman/prog/build/TrenchBroom/lib/KdLib/include/kd/ranges/detail/range_utils.h:23,
                 from /home/herman/prog/build/TrenchBroom/lib/KdLib/include/kd/ranges/zip_view.h:23:
/usr/lib/gcc/x86_64-pc-linux-gnu/15/include/g++-v15/bits/stl_algobase.h:234:5: note: candidate 1: ‘template<class _Tp> constexpr const _Tp& std::min(const _Tp&, const _Tp&)’
  234 |     min(const _Tp& __a, const _Tp& __b)
      |     ^~~
/usr/lib/gcc/x86_64-pc-linux-gnu/15/include/g++-v15/bits/stl_algobase.h:234:5: note: candidate expects 2 arguments, 1 provided
/usr/lib/gcc/x86_64-pc-linux-gnu/15/include/g++-v15/bits/stl_algobase.h:282:5: note: candidate 2: ‘template<class _Tp, class _Compare> constexpr const _Tp& std::min(const _Tp&, const _Tp&, _Compare)’
  282 |     min(const _Tp& __a, const _Tp& __b, _Compare __comp)
      |     ^~~
/usr/lib/gcc/x86_64-pc-linux-gnu/15/include/g++-v15/bits/stl_algobase.h:282:5: note: candidate expects 3 arguments, 1 provided
In file included from /usr/lib/gcc/x86_64-pc-linux-gnu/15/include/g++-v15/functional:55:
/usr/lib/gcc/x86_64-pc-linux-gnu/15/include/g++-v15/tuple: In instantiation of ‘constexpr decltype(auto) std::__apply_impl(_Fn&&, _Tuple&&, index_sequence<_Idx ...>) [with _Fn = kdl::ranges::zip_view<ranges::ref_view<const vector<int, allocator<int> > >, ranges::ref_view<const array<float, 3> > >::size() const requires (sized_range<Views> && ...)::<lambda(auto:21&& ...)>; _Tuple = const tuple<ranges::ref_view<const vector<int, allocator<int> > >, ranges::ref_view<const array<float, 3> > >&; long unsigned int ..._Idx = {0, 1}; index_sequence<_Idx ...> = integer_sequence<long unsigned int, 0, 1>]’:
/usr/lib/gcc/x86_64-pc-linux-gnu/15/include/g++-v15/tuple:2935:31:   required from ‘constexpr decltype(auto) std::apply(_Fn&&, _Tuple&&) [with _Fn = kdl::ranges::zip_view<ranges::ref_view<const vector<int, allocator<int> > >, ranges::ref_view<const array<float, 3> > >::size() const requires (sized_range<Views> && ...)::<lambda(auto:21&& ...)>; _Tuple = const tuple<ranges::ref_view<const vector<int, allocator<int> > >, ranges::ref_view<const array<float, 3> > >&]’
 2935 |       return std::__apply_impl(std::forward<_Fn>(__f),
      |              ~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~
 2936 |                                std::forward<_Tuple>(__t),
      |                                ~~~~~~~~~~~~~~~~~~~~~~~~~~
 2937 |                                _Indices{});
      |                                ~~~~~~~~~~~
/home/herman/prog/build/TrenchBroom/lib/KdLib/include/kd/ranges/zip_view.h:410:22:   required from ‘constexpr auto kdl::ranges::zip_view<Views>::size() const requires (sized_range<Views> && ...) [with Views = {std::ranges::ref_view<const std::vector<int, std::allocator<int> > >, std::ranges::ref_view<const std::array<float, 3> >}]’
  410 |     return std::apply(
      |            ~~~~~~~~~~^
  411 |       [](auto&&... b) { return std::min({std::ranges::size(b)...}); }, mBase);
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/herman/prog/build/TrenchBroom/lib/KdLib/test/src/ranges/tst_zip_view.cpp:171:7:   required from here
  171 |       CHECK(z.size() == 3);
      |             ~~~~~~^~
/usr/lib/gcc/x86_64-pc-linux-gnu/15/include/g++-v15/tuple:2920:27: error: no matching function for call to ‘__invoke(kdl::ranges::zip_view<std::ranges::ref_view<const std::vector<int, std::allocator<int> > >, std::ranges::ref_view<const std::array<float, 3> > >::size() const requires (sized_range<Views> && ...)::<lambda(auto:21&& ...)>, std::__tuple_element_t<0, std::tuple<std::ranges::ref_view<const std::vector<int, std::allocator<int> > >, std::ranges::ref_view<const std::array<float, 3> > > >&, std::__tuple_element_t<1, std::tuple<std::ranges::ref_view<const std::vector<int, std::allocator<int> > >, std::ranges::ref_view<const std::array<float, 3> > > >&)’
 2920 |       return std::__invoke(std::forward<_Fn>(__f),
      |              ~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~
 2921 |                            std::get<_Idx>(std::forward<_Tuple>(__t))...);
      |                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/lib/gcc/x86_64-pc-linux-gnu/15/include/g++-v15/tuple:2920:27: note: there is 1 candidate
In file included from /usr/lib/gcc/x86_64-pc-linux-gnu/15/include/g++-v15/tuple:43:
/usr/lib/gcc/x86_64-pc-linux-gnu/15/include/g++-v15/bits/invoke.h:92:5: note: candidate 1: ‘template<class _Callable, class ... _Args> constexpr typename std::__invoke_result<_Functor, _ArgTypes>::type std::__invoke(_Callable&&, _Args&& ...)’
   92 |     __invoke(_Callable&& __fn, _Args&&... __args)
      |     ^~~~~~~~
/usr/lib/gcc/x86_64-pc-linux-gnu/15/include/g++-v15/bits/invoke.h:92:5: note: template argument deduction/substitution failed:
/usr/lib/gcc/x86_64-pc-linux-gnu/15/include/g++-v15/bits/invoke.h: In substitution of ‘template<class _Callable, class ... _Args> constexpr typename std::__invoke_result<_Functor, _ArgTypes>::type std::__invoke(_Callable&&, _Args&& ...) [with _Callable = kdl::ranges::zip_view<std::ranges::ref_view<const std::vector<int, std::allocator<int> > >, std::ranges::ref_view<const std::array<float, 3> > >::size() const requires (sized_range<Views> && ...)::<lambda(auto:21&& ...)>; _Args = {const std::ranges::ref_view<const std::vector<int, std::allocator<int> > >&, const std::ranges::ref_view<const std::array<float, 3> >&}]’:
/usr/lib/gcc/x86_64-pc-linux-gnu/15/include/g++-v15/tuple:2920:27:   required from ‘constexpr decltype(auto) std::__apply_impl(_Fn&&, _Tuple&&, index_sequence<_Idx ...>) [with _Fn = kdl::ranges::zip_view<ranges::ref_view<const vector<int, allocator<int> > >, ranges::ref_view<const array<float, 3> > >::size() const requires (sized_range<Views> && ...)::<lambda(auto:21&& ...)>; _Tuple = const tuple<ranges::ref_view<const vector<int, allocator<int> > >, ranges::ref_view<const array<float, 3> > >&; long unsigned int ..._Idx = {0, 1}; index_sequence<_Idx ...> = integer_sequence<long unsigned int, 0, 1>]’
 2920 |       return std::__invoke(std::forward<_Fn>(__f),
      |              ~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~
 2921 |                            std::get<_Idx>(std::forward<_Tuple>(__t))...);
      |                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/lib/gcc/x86_64-pc-linux-gnu/15/include/g++-v15/tuple:2935:31:   required from ‘constexpr decltype(auto) std::apply(_Fn&&, _Tuple&&) [with _Fn = kdl::ranges::zip_view<ranges::ref_view<const vector<int, allocator<int> > >, ranges::ref_view<const array<float, 3> > >::size() const requires (sized_range<Views> && ...)::<lambda(auto:21&& ...)>; _Tuple = const tuple<ranges::ref_view<const vector<int, allocator<int> > >, ranges::ref_view<const array<float, 3> > >&]’
 2935 |       return std::__apply_impl(std::forward<_Fn>(__f),
      |              ~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~
 2936 |                                std::forward<_Tuple>(__t),
      |                                ~~~~~~~~~~~~~~~~~~~~~~~~~~
 2937 |                                _Indices{});
      |                                ~~~~~~~~~~~
/home/herman/prog/build/TrenchBroom/lib/KdLib/include/kd/ranges/zip_view.h:410:22:   required from ‘constexpr auto kdl::ranges::zip_view<Views>::size() const requires (sized_range<Views> && ...) [with Views = {std::ranges::ref_view<const std::vector<int, std::allocator<int> > >, std::ranges::ref_view<const std::array<float, 3> >}]’
  410 |     return std::apply(
      |            ~~~~~~~~~~^
  411 |       [](auto&&... b) { return std::min({std::ranges::size(b)...}); }, mBase);
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/herman/prog/build/TrenchBroom/lib/KdLib/test/src/ranges/tst_zip_view.cpp:171:7:   required from here
  171 |       CHECK(z.size() == 3);
      |             ~~~~~~^~
/usr/lib/gcc/x86_64-pc-linux-gnu/15/include/g++-v15/bits/invoke.h:92:5: error: no type named ‘type’ in ‘struct std::__invoke_result<kdl::ranges::zip_view<std::ranges::ref_view<const std::vector<int, std::allocator<int> > >, std::ranges::ref_view<const std::array<float, 3> > >::size() const requires (sized_range<Views> && ...)::<lambda(auto:21&& ...)>, const std::ranges::ref_view<const std::vector<int, std::allocator<int> > >&, const std::ranges::ref_view<const std::array<float, 3> >&>’
   92 |     __invoke(_Callable&& __fn, _Args&&... __args)
      |     ^~~~~~~~
make[2]: *** [lib/KdLib/test/CMakeFiles/KdLibTest.dir/build.make:275: lib/KdLib/test/CMakeFiles/KdLibTest.dir/src/ranges/tst_zip_view.cpp.o] Error 1
make[2]: Leaving directory '/media/prog/build/TrenchBroom/build'
make[1]: *** [CMakeFiles/Makefile2:908: lib/KdLib/test/CMakeFiles/KdLibTest.dir/all] Error 2
make[1]: Leaving directory '/media/prog/build/TrenchBroom/build'
make: *** [Makefile:156: all] Error 2
```

--- a/lib/KdLib/include/kd/ranges/zip_view.h
+++ b/lib/KdLib/include/kd/ranges/zip_view.h
@@ -23,6 +23,7 @@
 #include "detail/range_utils.h"
 #include "detail/tuple_common_reference.h" // IWYU pragma: keep
 
+#include <algorithm>
 #include <ranges>
 #include <tuple>
 

